wayfire.ini.default

```diff
@@ -131,15 +127,28 @@ mode = cascade
 # show the current workspace row as a cube
 [cube]
 activate = <ctrl> <alt> BTN_LEFT
-background = 0.1 0.1 0.1 1.0
+# set deform to 1 or 2 if you have OpenGL ES 3.2
```

Comment:
It should be added:
> you can set deform ..

As it is currently stated, this is telling the user they should always use 1 or 2 if you have 3.2